### ISSUE TITLE
Text spammer not working

### ISSUE THREAD

**Issue by pecatum** [Merubokkusu/discord-spam-bots#92]
Getting no error. But not working. Repeating DM spammer started text. 
![image](https://user-images.githubusercontent.com/77746578/110564666-4b969200-815e-11eb-8b98-ac38f662ec03.png)

**Comment by Merubokkusu** [comment]
Set discumLog to true in the config and post the results

**Comment by pecatum** [comment]
![image](https://user-images.githubusercontent.com/77746578/110565383-77fede00-815f-11eb-80bd-bb84b8a69d9e.png)
repeating dm spammer started text makes that hard but i finally catch the exact point when it started repeating.
![image](https://user-images.githubusercontent.com/77746578/110565704-ee034500-815f-11eb-8063-84643a7bcfe8.png)

**Comment by Merubokkusu** [comment]
Weird.

Python version?
Discum up-to date?

**Comment by pecatum** [comment]
py 3.7.9 
Discum and others fresh installed yesterday. After proxy fix ı assume 1.1.0.

**Comment by Merubokkusu** [comment]
Using proxies?

**Comment by pecatum** [comment]
No, but I'm using only 4 tokens.  All of them email verified and created yesterday. My IP is okay also my main DC account is working fine.

Edit;

I just created clean install of server 2019. and installed py 3.7.8, nodejs, git then cloned fresh installed all requirements then got another weird error lol. Believe me its realy weird.

![resim](https://user-images.githubusercontent.com/77746578/110715247-eb632700-8215-11eb-8efe-4298279ddcc0.png)
well i checked all;

![resim](https://user-images.githubusercontent.com/77746578/110715348-1cdbf280-8216-11eb-8a14-6aa7e3ce604f.png)

now its even weird. its working on my main pc (just re downloaded changed files and merged with old folder) and not working on fresh installed operating system.

**Comment by Merubokkusu** [comment]
Weird, have you tried a reboot?
I remember having a similar issue

**Comment by Aljoberg** [comment]
Yeah, i have the same issue. I'm on Repl.it, which doesn't change anything. I am getting the spammed text just like this guy. Can someone help me?
